test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -2317,7 +2317,11 @@ private static void initializeSuiteScope() throws Exception {
         assert INSTANCE == null;
         if (isSuiteScopedTest(targetClass)) {
             // note we need to do this this way to make sure this is reproducible
-            INSTANCE = (OpenSearchIntegTestCase) targetClass.getConstructor().newInstance();
+            if (targetClass.getAnnotation(SuiteScopeTestCase.class).isParameterized()) {
```

Comment:
👍

Reply:
Could we just use reflection to check if the test class has a constructor that takes a single `Settings` parameter and if so supply empty settings to it?

Reply:
@reta @andrross thanks for the suggestions, i have updated the code to check this by reflection.